calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorActivity.kt

```diff
@@ -13,7 +13,8 @@ import kotlinx.coroutines.launch
 
 class CalculatorActivity : AppCompatActivity() {
     private lateinit var binding: ActivityCalculatorBinding
-    private val calculatorVM by viewModels<CalculatorViewModel>()
+    private val calculatorVM by viewModels<CalculatorViewModel> { CalculatorViewModelFactory(this) }
+    private val calculatorHistoryAdapter by lazy { CalculatorHistoryAdapter() }
```

Comment:
```suggestion
    private val calculatorHistoryAdapter = CalculatorHistoryAdapter()
```
지연초기화는 주로 초기화 시점이 라이프싸이클에 영향을 받는 경우에 사용합니다.
현재 CalculatorHistoryAdapter는 라이프싸이클의 영향 없이 객체 생성이 가능합니다.

지연 초기화는 사용하는 시점에 초기화를 하기 때문에 초기 진입속도 향상에 기여할 수 있지만, 매번 초기화 시점이 다르고 성능 측정이 어렵습니다.
해서 가급적이면 일반 초기화를 사용하되, 필요에 따라 지연 초기화를 활용해보면 어떨까요?

Reply:
[8074e08](https://github.com/next-step/android-calculator-mvvm/pull/113/commits/8074e082d20dfcc3e4b203df870c7d294b44f5c8)

지연초기화는 주로 초기화 시점이 라이프싸이클에 영향을 받는 경우
기억하겠습니다

Reply:
가령 Room과 같이 초기화 시 Context가 필요한 경우에는, Activity 혹은 Application이 생성되고 난 후에 초기화가 가능하기에, 일반 초기화를 사용할 수가 없습니다.

Reply:
적절한 예시 감사합니다~!